### PR TITLE
[BAEL-16003] Added libraries-data-2

### PR DESCRIPTION
**Pull request by sjmillington**
- Copied all but joda-time module into libraries-data-2.

### CODE REVIEW

File: libraries-data-2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>libraries-http</artifactId>
```
Comment (lor6) [inline libraries-data-2/pom.xml]: should be libraries-data-2

File: libraries-data-2/src/main/com/baeldung/smooks/converter/OrderConverter.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import org.milyn.payload.StringResult;
+import org.xml.sax.SAXException;
+
+import javax.xml.transform.stream.StreamSource;
+import java.io.IOException;
+
+public class OrderConverter {
+
+    public Order convertOrderXMLToOrderObject(String path) throws IOException, SAXException {
+        Smooks smooks = new Smooks(OrderConverter.class.getResourceAsStream("/smooks/smooks-mapping.xml"));
```
Comment (lor6) [inline libraries-data-2/src/main/com/baeldung/smooks/converter/OrderConverter.java]: this file is missing in the PR

File: libraries-data-2/src/test/com/baeldung/smooks/converter/SmooksIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    private static final String EDIFACT_MESSAGE = "UNA:+.? '" + System.lineSeparator() + "UNH+771+IN_PROGRESS+2018-01-14'" + System.lineSeparator() + "CTA+CompanyX+1234567'" + System.lineSeparator() + "LIN+1+PX1234+9.99'" + System.lineSeparator()
+        + "LIN+2+RX990+120.32'" + System.lineSeparator();
+    private static final String EMAIL_MESSAGE = "Hi," + System.lineSeparator() + "Order number #771 created on 2018-01-14 is currently in IN_PROGRESS status." + System.lineSeparator() + "Consider contact supplier \"CompanyX\"  with phone number: \"1234567\"."
+        + System.lineSeparator() + "Order items:" + System.lineSeparator() + "1  X PX1234 (total price 9.99)" + System.lineSeparator() + "2  X RX990 (total price 240.64)" + System.lineSeparator();
+
+    @Test
+    public void givenOrderXML_whenConvert_thenPOJOsConstructedCorrectly() throws Exception {
+
+        OrderConverter xmlToJavaOrderConverter = new OrderConverter();
+        Order order = xmlToJavaOrderConverter.convertOrderXMLToOrderObject("/smooks/order.xml");
```
Comment (lor6) [inline libraries-data-2/src/test/com/baeldung/smooks/converter/SmooksIntegrationTest.java]: this file is missing in the PR

File: libraries-data-2/pom.xml (hunk 1 above)
Comment (sjmillington) [reply libraries-data-2/pom.xml]: added spooks resources into libraries-data-2

File: libraries-data-2/src/main/com/baeldung/smooks/converter/OrderConverter.java (hunk 2 above)
Comment (sjmillington) [reply libraries-data-2/src/main/com/baeldung/smooks/converter/OrderConverter.java]: added spooks resources into libraries-data-2

File: libraries-data-2/src/test/com/baeldung/smooks/converter/SmooksIntegrationTest.java (hunk 3 above)
Comment (sjmillington) [reply libraries-data-2/src/test/com/baeldung/smooks/converter/SmooksIntegrationTest.java]: added spooks resources into libraries-data-2

**Pull request merged**